### ISSUE TITLE
Build error: "dx: not found"

### ISSUE THREAD

**Issue by MaxGyver83** [Julow/Unexpected-Keyboard#54]
When I build using `make`, I get this error:

```
...
/home/max/Android/Sdk/build-tools/32.0.0/dx --dex --output="_build/classes.dex" _build/class
/bin/sh: 1: /home/max/Android/Sdk/build-tools/32.0.0/dx: not found
make: *** [Makefile:105: _build/classes.dex] Error 127
```

The problem is that the Makefile chooses the newest Android build-tools version available:

```Makefile
ANDROID_BUILD_TOOLS = $(lastword $(sort $(wildcard $(ANDROID_HOME)/build-tools/*)))
```

But `dx` is missing in SDK 31 and 32 (at least on my system):

```sh
~/Android/Sdk ▶ ll ~/Android/Sdk/build-tools/
insgesamt 20K
drwxrwxr-x 5 max max 4,0K Dez 22 10:43 28.0.3/
drwxrwxr-x 5 max max 4,0K Jul 30  2021 29.0.3/
drwxrwxr-x 6 max max 4,0K Feb  1 22:25 30.0.3/
drwxrwxr-x 6 max max 4,0K Jul 30  2021 31.0.0/
drwxrwxr-x 6 max max 4,0K Dez 22 10:45 32.0.0/
```

```sh
~/Android/Sdk ▶ fd 'dx$'
build-tools/28.0.3/dx
build-tools/29.0.3/dx
build-tools/30.0.3/dx
```

[CONTRIBUTING.md](https://github.com/Julow/Unexpected-Keyboard/blob/master/CONTRIBUTING.md) says, build tools version `30.0.3` is a dependency. I guess the Makefile shouldn't pick any newer versions.

For me it works when I hardcode version `30.0.3`:

```diff
diff --git a/Makefile b/Makefile
index 6b873e4..532be36 100644
--- a/Makefile
+++ b/Makefile
@@ -27,7 +27,8 @@ ifndef ANDROID_HOME
 $(error ANDROID_HOME not set)
 endif

-ANDROID_BUILD_TOOLS = $(lastword $(sort $(wildcard $(ANDROID_HOME)/build-tools/*)))
+ANDROID_BUILD_TOOLS = $(ANDROID_HOME)/build-tools/30.0.3
```

Does anybody know, why there is no `dx` in the latest build-tools?

**Comment by Julow** [comment]
I've noticed this and that's why I'm stuck on `30.0.3` for now. I think `dx` has been replaced with [d8](https://developer.android.com/studio/command-line/d8).
In an ideal world, the Makefile would take whatever is the latest build tool but here a tool has been removed. I'll fix this soon, in the meantime, you can continue with the hardcoded version.

Thanks a lot for all the contributions ! I'm a bit busy at the moment but I'll merge everything this weekend.

**Comment by Julow** [comment]
Should be fixed (75c7367)
